sites/hashdev/src/_pages/blog/10_error-stack-update-0-2.mdx

```diff
@@ -0,0 +1,197 @@
+---
+authorPhoto: blog/authors/tim-diekmann.jpg
+postPhoto: blog/10_error-stack-update-0-2/Cover.jpg
+title: "Announcing error-stack v0.2"
+subtitle: "Redesigned and customizable outputs, related errors and multiple sources, and much more"
+author: "Tim Diekmann"
+jobTitle: "Platform Engineer, HASH"
+date: "2022-10-03"
+---
+
+Back in June [we announced the first version](https://hash.dev/blog/announcing-error-stack) of `error-stack`, our context-aware error library. We developed this to help address a lot of challenges we encountered when implementing error handling in our [simulation engine]. In the time since then we’ve started using it in our entity-graph query layer that we’re building as a datastore for [HASH]. Across that development we’ve learned a lot of lessons, and in joint efforts with open-source contributors we’re ready to release a new version of the library.
+
+Since our last release we’ve been fixing user-reported bugs, reviewing open-source contributor’s pull requests, and adding new features. [Version 0.2](https://github.com/hashintel/hash/tree/error-stack%400.2.0/packages/libs/error-stack) is available now and includes important changes from `std`, a completely redesigned output with new customization options, support for related errors and multiple error sources, compatibility with `anyhow` and `eyre`, and a number of other improvements.
```

Comment:
@TimDiekmann Do we want to link to this GitHub page, or the. `crates.io` listing?

Reply:
I changed that link to point to crates.io and added a link to GitHub as well - similar as we did in the initial blog post: f171bbbc94a6fec6844d4317cbe541562d881e76